themes/ddbasic/sass/base/form.scss

```diff
@@ -317,6 +325,16 @@ select {
   }
 }
 
+select[multiple="multiple"] {
+  position: relative;
```

Comment:
Why do we need `position: relative` here? As far as I can tell the element itself is not positioned using `top`, `left` etc. and child elements are not positioned either. If we want to reset the position `absolute` from above then we should use `position: static;`.